uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/sub-controllers/sick-request.controller.js

```diff
@@ -6,25 +6,26 @@ define([
 ], function (_, controllers) {
   controllers.controller('SicknessRequestCtrl', SicknessRequestCtrl);
 
-  SicknessRequestCtrl.$inject = ['$log', '$q', 'api.optionGroup', 'parentCtrl'];
+  SicknessRequestCtrl.$inject = ['$controller', '$log', '$q', 'api.optionGroup', 'parentCtrl'];
 
-  function SicknessRequestCtrl ($log, $q, OptionGroup, parentCtrl) {
+  function SicknessRequestCtrl ($controller, $log, $q, OptionGroup, parentCtrl) {
     $log.debug('SicknessRequestCtrl');
-    var vm = parentCtrl;
+    // We need to extend parent controller with
```

Comment:
"we need" doesn't say anything useful. Some time from now we will be wondering why we "needed" it. Was it a workaround? Was it something intentional or was it an ad-hoc solution?

A shorter, even more generic comment, can quickly get rid of any doubt, showing that this was intentional and planned, example:

```js
// Shares basic logic with the the leave controller
$controller('LeaveRequestCtrl', { parentCtrl: parentCtrl });
```

Reply:
Done.